### ISSUE TITLE
Improve CRA bootstrap “Next Steps"

### ISSUE THREAD

**Issue by thedavidprice** [redwoodjs/redwood#200]
Add information about:
- website (docs) and tutorial with link
- main rw commands
- cd /dir
- `git init && git add .` then first commit

Example:
```
Created git commit.

Success! Created my-app at /Users/price/Repos/my-app
Inside that directory, you can run several commands:

  yarn start
    Starts the development server.

  yarn build
    Bundles the app into static files for production.

  yarn test
    Starts the test runner.

  yarn eject
    Removes this tool and copies build dependencies, configuration files
    and scripts into the app directory. If you do this, you can’t go back!

We suggest that you begin by typing:

  cd my-app
  yarn start

Happy hacking!
```

**Comment by cannikin** [comment]
HMMM should we be telling people to use `yarn start` instead of `yarn rw dev`?

**Comment by thedavidprice** [comment]
I like the consistency of always using `yarn rw …` Vote to leave as is.

I do wish it was just `rw …` and to be honest have thought about aliasing many times ;)

**Comment by thedavidprice** [comment]
closed in favor of #909 909
